src/main/java/htsjdk/variant/utils/SAMSequenceDictionaryExtractor.java

```diff
@@ -111,7 +144,10 @@ SAMSequenceDictionary extractDictionary(final File intervalList) {
             applicableExtensions = extensions;
         }
 
+        /** @deprecated in favor of extractDictionary(final Path file) */
```

Comment:
Better to use `@link` here so it shows up as a hyperlink, e.g. 
```
/**
  * @deprecated use  {@link #extractDictionary(Path)} instead
  */
```

Reply:
done.